eng/Versions.props

```diff
@@ -32,5 +32,20 @@
     <SystemSecurityPrincipalWindowsVersion>5.0.0</SystemSecurityPrincipalWindowsVersion>
     <!-- wcf -->
     <SystemServiceModelVersion>8.1.2</SystemServiceModelVersion>
+    <!-- wix -->
+    <MicrosoftWixToolsetSdkVersion>5.0.2-dotnet.2737382</MicrosoftWixToolsetSdkVersion>
+    
+    <!-- Individual WiX component versions (following runtime repo pattern) -->
+    <MicrosoftWixVersion>5.0.2-dotnet.2737382</MicrosoftWixVersion>
```

Comment:
You should be able to use the Arcade property `MicrosoftWixToolsetSdkVersion` for these (that's what aspnetcore does)